coaster/logger.py

```diff
@@ -109,6 +110,10 @@ def formatException(self, ei):
             print("App context:", file=sio)
             pprint_with_indent(vars(g), sio)
 
+        print('\n----------\n', file=sio)
+        print("Current user:", file=sio)
+        pprint_with_indent(vars(current_user), sio)
```

Comment:
This currently renders as `{'self': <User MFLi1lfRR2eTpqRj1fXe8g (jace) "Kiran Jonnalagadda">}`. If we use `vars(current_user.self)`, we get too much unnecessary detail on the user object. Rendering `current_user` alone will also capture other variables within it as it grows in scope.